### DISCUSSION TITLE
names of individual sub-switches

### DISCUSSION THREAD

**Discussion by Strawqens** [merdok/homebridge-miot#647] [General]
I would like to translate the names of individual sub-switches, but I can't find them in the JSON file. Does anyone know where to edit them?

**Comment by merdok** [comment]
They are auto generated, you should use the HomeKit ui to rename the switches

**Reply by Strawqens** [reply]
However, after every Homebridge reset, the names revert to the old ones. There's another issue as well: I’m having trouble turning off Yeelight ceiling lamps. It takes some effort to turn them off, or I have to do it in night mode, because simply sliding the control doesn’t work. It would also be great to see an option for adjusting the light temperature, as it is in the Yeelighter plugin. By the way, I don’t understand why Yeelight hasn’t added color control to HomeKit yet...

**Reply by merdok** [reply]
Yeah well this is a known issues... at some point i plan to fix the name persistence but that will take some effort on the development side so i am not sure when i will do that.

**Comment by Strawqens** [comment]
I will be waiting and keeping my fingers crossed! Nevertheless, very good job! Pozdro ;)
